src/app/[lang]/partners/page.tsx

```diff
@@ -11,7 +11,7 @@ export default function Partners() {
   const partners = dict.partners;
 
   return (
-    <main className="flex flex-col gap-8 sm:gap-12">
+    <main className="flex flex-col gap-8 sm:gap-12 selection:bg-primary selection:text-foundation">
```

Comment:
You can add this tailwind classes to the layout so they are applied to all pages by default, and only apply them on different places when you want a different select color, for example in the navbar.